master/cluster.go

```diff
@@ -1289,6 +1289,25 @@ func (c *Cluster) markDeleteVol(name, authKey string, force bool) (err error) {
 		return proto.ErrVolNotExists
 	}
 
+	vol.mpsLock.RLock()
```

Comment:
it's better to add a switch config to control whether to check `dentryCnt == 0`, eg, add a config key `forceDelVol`, when it's true, not check dentry count,  `false` will check. And set default value is true to keep consistent with the original deletion plan.
And it's also will be better to add a config key like `delDentryCnt`, when dentyCnt < `delDentryCnt`, vol can be deleted.

Reply:
if default value of forceDelVol  is true , risk of accidental deletion still exists.  I think false is the resonable default value.   And i will fix the  ci-test-unit and ci-test-s3 error

Reply:
In fact, it will be better to add a switch in the configuration file of master, when the switch is turned on, the entire volume can be deleted directly, and when it is not opened, only the volume with the number of dentries less than the specified value can be deleted, which is more secure and avoids misoperation.

Reply:
shoud i implement this requirements in this issue ?
1) add _volFroceDeletion_ in master configuration file (default false)
2) add _volDeletionDentryThreshold_ in master configuration file (default 10)

if so  i will undo the cfs-cli  and master AdminDeleteVol interface changes

Reply:
yes, it's better to implement the both requirements, but `volFroceDelete` can be set default true, so it will keep consistent with the versions before.

Reply:
i reopened a pr to resolve this